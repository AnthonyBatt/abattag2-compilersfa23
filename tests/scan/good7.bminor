These will all read as Identifiers
