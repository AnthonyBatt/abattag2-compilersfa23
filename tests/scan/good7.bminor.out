TOKEN_ID                  These
TOKEN_ID                  will
TOKEN_ID                  all
TOKEN_ID                  read
TOKEN_ID                  as
TOKEN_ID                  Identifiers
